backend.py

```diff
@@ -212,6 +238,44 @@ def submit_wifi_form():
         })
         DB.commit()
 
+        # Send a mail, with config in settings.ini. Fail silently.
+        try:
+            config = read_config()
+            default_subject = "[wifi-with-me] nouvelle demande de {}".format(d.get('name'))
+            SUBJECT = default_subject
+            FROM = config.get("from")
+            TO = config.get("to")
+            HOST = config.get('smtp_host')
+
+            infos = """
+            - son mail: {}\n
+            - son tel: {}\n
+            """.format(d.get('email'), d.get('phone'))
+
+            BODY = """From: <{}>
+    To: To Person <{}>
```

Comment:
Pourquoi ne pas juste mettre l'adresse mail plutôt que cette "To Person" ?

Je vois un souci : la partie header du mail va débuter par 4 espaces systématiquement. Je ne sais pas si tous les MTA apprécieront, en tout cas c'est peu courant.